server/models/CardCoverPlugin.js

```diff
@@ -0,0 +1,89 @@
+const { Schema, model } = require("mongoose")
+const BasePlugin = require("./BasePlugin")
+const { v2: cloudinary } = require("cloudinary")
+
+const CardCoverPluginSchema = BasePlugin.discriminator("CardCoverPlugin", new Schema({
+  imageUrl: {
+    type: String,
+    required: true
+  },
+  imagePublicId: {
+    type: String,
+    required: true
+  }
+}, {timestamps: true}))
+
+CardCoverPluginSchema.methods.get = async function({params}) {
+  if(!params) {
+    throw new Error("Not a valid request!")
+  }
+  return await this.model("CardCoverPlugin").findById(params.coverId)

```

Comment:
So, we're actually going to be enforcing a response structure. 

`{  status code, data/message }`

200 responses should be the following `{ response: 200, data: <Cover data> }`
Anything else -> `{  response: XXX, message: "Any message" }`

Reply:
Please update in all your action handlers.